anomalib/core/model/k_center_greedy.py

```diff
@@ -32,31 +30,19 @@ class KCenterGreedy:
         torch.Size([219, 1536])
     """
 
-    def __init__(self, model: SparseRandomProjection, embedding: Tensor, sampling_ratio: float) -> None:
-        self.model = model
+    def __init__(self, embedding: Tensor, sampling_ratio: float) -> None:
         self.embedding = embedding
         self.coreset_size = int(embedding.shape[0] * sampling_ratio)
+        self.model = random_projection.SparseRandomProjection(eps=0.9)
```

Comment:
The reason for using Torch SparseRandomProjection implementation was to be able to export the model to OpenVINO / ONNX. I was wondering if it is possible to do the same thing with sklearn SparseRandomProjection.

Reply:
@samet-akcay, I have verified that the original Torch implementation produces same result as the sklearn one. If the conversion is not possible with sklearn, it might be easier to switch back to the Torch implementation. I made this change just because I thought the implementation is cleaner with less code.

Reply:
good to hear, thanks!

Reply:
To convert to ONNX/OpenVINO and to make the computation faster on GPU, we could then use Torch implementation. We could move `SparseRandomProjection` to `anomalib.core.model`, which would de-clutter PatchCore implementation itself. We could discuss this further